src/App.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
```

Comment:
- [ ] Please kindly fix your linters errors instead of disabling them. Fix linters help us to write clean and professional codebase 😉